src/R/Components/Impl/History/Implementation/RHistoryProvider.cs

```diff
@@ -24,7 +24,12 @@ internal class RHistoryProvider : IRHistoryProvider {
         private readonly Dictionary<ITextBuffer, IRHistory> _histories;
 
         [ImportingConstructor]
-        public RHistoryProvider(ITextBufferFactoryService textBufferFactory, IContentTypeRegistryService contentTypeRegistryService, IEditorOperationsFactoryService editorOperationsFactory, IRtfBuilderService rtfBuilderService, ITextSearchService2 textSearchService, IRSettings settings) {
+        public RHistoryProvider(ITextBufferFactoryService textBufferFactory 
+            , IContentTypeRegistryService contentTypeRegistryService
+            , IEditorOperationsFactoryService editorOperationsFactory
+            , IRtfBuilderService rtfBuilderService
```

Comment:
Yes

---
In reply to: [81234076](https://github.com/Microsoft/RTVS/pull/2408#discussion_r81234076) [](ancestors = 81234076)